.github/workflows/pythonpackage.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     - name: Test with pytest
       run: |
         pip install pytest
-        pytest -v
+        RECIRQ_CHESS_TEST_SEED=789 pytest -v
```

Comment:
Maybe add a comment here to explain what this is for, in case people come here for a different part of reCirq.

Reply:
Done.